src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -4139,6 +4139,109 @@ theorem sdiv_self {x : BitVec w} :
       rcases x.msb with msb | msb <;> simp
     · rcases x.msb with msb | msb <;> simp [h]
 
+/-- Unsigned division never overflows. -/
+theorem toNat_udiv_lt {w : Nat} {x y : BitVec w} :
+    x.toNat / y.toNat < 2 ^ w := by
```

Comment:
```suggestion
theorem toNat_div_toNat_lt {w : Nat} {x y : BitVec w} :
    x.toNat / y.toNat < 2 ^ w := by
```

- AFAIU `/` on type Nat is just named `div`.
- There are two toNat on the LHS. AFAIU they should both be used in the name.

Reply:
okay, so should I also rename all the other ones to `toInt_ediv_toInt_*`?

Reply:
Yes, I just am heading for breakfast but was interrupted in my review. Can you carefully look over the names. I will make a pass later today.

Reply:
done for all `BitVec.toInt_ediv_*` theorems